parents_page.html

```diff
@@ -0,0 +1,32 @@
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+<style>
+    h1 {
+        text-align: center;
+    }
+    p {
+        text-align: center;
+        font-size: 20px;
+    }
+</style>
+
+<h1>The Kadri Parents</h1>
+<div class="row container">
+        <div class="col-md-3">
+            <p></p>
+        </div>
+        <div class="col-md-6">
+            <img src="https://images.squarespace-cdn.com/content/v1/53d25f8be4b00410ab98b3c0/1420323510729-ZCR7YFVRMOC0UB4PBPAN/ke17ZwdGBToddI8pDm48kJHrlBiN1PjqnM1-Uxa1oz5Zw-zPPgdn4jUwVcJE1ZvWQUxwkmyExglNqGp0IvTJZUJFbgE-7XRK3dMEBRBhUpx6WvZJQsPBLsoI_bgKee14AQPrILwBR6Jrkp2DZJQMyGaIlcqPBK5LOGrkjfMLEp0/Lamar+And+Ronnie+Tyler+Black+And+Married+With+Kids" alt="">
+        </div>
+        <div class="col-md-3">
+            <p></p>
+        </div>
+    </div>
+    <div class="row container">
+            <div class="col-md-12">
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Tristique senectus et netus et malesuada fames ac turpis egestas. Euismod in pellentesque massa placerat duis ultricies lacus sed. Vestibulum mattis ullamcorper velit sed ullamcorper morbi tincidunt. Quam pellentesque nec nam aliquam sem et. Id ornare arcu odio ut sem nulla. Facilisi nullam vehicula ipsum a arcu cursus. Euismod quis viverra nibh cras pulvinar mattis nunc sed. Consectetur adipiscing elit ut aliquam purus sit amet luctus venenatis. In fermentum et sollicitudin ac orci phasellus egestas. Quam adipiscing vitae proin sagittis. Aliquet risus feugiat in ante metus dictum at tempor. Mauris vitae ultricies leo integer malesuada nunc vel. Nulla facilisi morbi tempus iaculis urna id volutpat lacus laoreet. Sit amet nisl suscipit adipiscing bibendum est ultricies integer quis. Luctus venenatis lectus magna fringilla urna porttitor. Feugiat nisl pretium fusce id velit ut tortor pretium. Vulputate sapien nec sagittis aliquam malesuada bibendum arcu. Mauris pellentesque pulvinar pellentesque habitant morbi tristique senectus et.
```

Comment:
Add a new line at the bottom of the code. Also, put actual info about the parents.
Like name and profession. Not age.